modules/genomic_browser/ajax/genomic_file_upload.php

```diff
@@ -120,15 +167,28 @@ function moveFileToFS(&$fileToUpload)
 
     $config           = NDB_Config::singleton();
     $genomic_data_dir = $config->getSetting('GenomicDataPath');
-    $DB =& Database::singleton();
-    reportProgress(40, "Copying file to $genomic_data_dir ");
-    if (move_uploaded_file(
+    $validPreconditions = true;
+
+    // update ui to show we are trying to move the file
+    reportProgress(98, "Copying file to $genomic_data_dir ");
```

Comment:
Do we want to show the user the full path on the server? Why not just "Copying file.."?